README.md

```diff
@@ -1,113 +1,101 @@
-# The PrimeDAO’s Home for IDOs
+# Prime Deals dApp
+Prime Deals is an An Interface for DAO to DAO interactions, such as token swaps, co-liquidity provision, and joint venture formation. An introduction to Prime Deals [TODO: ADD LINK](link-to-introduction).
 
-This is the [PrimeDAO](primedao.eth.link) web interface enabling the creation of and interaction with IDOs.
+This prime-deals-dapp repository contains in its master branch the web user interface for the Prime Deals website that is deployed to https://deals.prime.xyz.
 
-## Technical Description
+## Development
 
-This project is bootstrapped by [aurelia-cli](https://github.com/aurelia/cli).
+### Prerequisites
+Make sure you have [node.js version >= 14.11.0 < 15](https://nodejs.org/en/)
```

Comment:
This is not what the package.json file says.  Can you update that as part of this PR?

Reply:
Should be < 15, or <= 15?

Reply:
> Should be < 15, or <= 15?

Following @paweljanicki proposal, it should be `<15`, so no major upgrades for node will be done.